ubuntu/common/install_nvidiagpudriver.sh

```diff
@@ -7,7 +7,7 @@ CHECKSUM=$2
 
 # Reference - https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#ubuntu-installation
 # Install Cuda
-NVIDIA_VERSION="510.73.08"
+NVIDIA_VERSION="510.85.02"
```

Comment:
Should we update fabric manager also? [install_nvidia_fabric_manager.sh](https://github.com/Azure/azhpc-images/blob/master/ubuntu/common/install_nvidia_fabric_manager.sh)

Also, are you planning to have a separate PR for the LTS image?

Reply:
Done, yes I will create a separate PR for the LTS image.

Reply:
It turns out I needed to update nccl to get the latest version of the nccl-tests to work. Is that acceptable?